libs/features/mas/web-components/test/merch-card.ccd-suggested.test.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Merch Card CCD Suggested Web Component demo page</title>
+    <script>
+        window.process = {
+            env: {},
+        };
+    </script>
+    <style>
+        main {
+            display: flex;
+            align-items: center;
+            gap: 20px;
+            flex-direction: column;
+            padding-top: 80px;
+            width: 100%;
+            height: 100%;
+        }
+        merch-card {
+          margin-bottom: 20px;
+        }
+    </style>
+</head>
+
+<body>
+    <script type="module" src="./merch-card.ccd-suggested.test.html.js"></script>
+    <main>
+      <sp-theme theme="spectrum" color="light" scale="medium" dir="ltr">
+          <merch-card class="" tabindex="0" variant="ccd-suggested" filters="" types="">
+            <sp-status-light size="l" variant="info" dir="ltr"></sp-status-light>
+            <merch-icon
+              slot="icons"
+              src="https://www.adobe.com/content/dam/shared/images/product-icons/svg/photoshop.svg"
+              alt=""
+              href=""
+              size="l"
+              >undefined</merch-icon
+            >
+            <merch-icon
+              slot="icons"
+              src="https://www.adobe.com/content/dam/shared/images/product-icons/svg/lightroom.svg"
+              alt=""
+              href=""
+              size="l"
+              >undefined</merch-icon
+            >
+            <p slot="detail-m">YOUR EXPIRED PLAN</p>
+            <p slot="heading-xs"><strong>Photography plan (20GB)</strong></p>
+            <p slot="body-xs">
+              Creative Cloud Photography plan.<br>
+              Starting at US$19.99/mo
+            </p>
+            <div slot="price">
+              <span
+          is="inline-price"
+          data-display-old-price="true"
+          data-display-per-unit="false"
+          data-promotion-code=""
+          data-quantity="1"
+          data-template="price"
+          data-wcs-osi="r_JXAnlFI7xD6FxWKl2ODvZriLYBoSL701Kd1hRyhe8"
+          class="placeholder-resolved"
+          ><span class="price" aria-label="US$35.99 per month"
```

Comment:
please avoid copying the markup of an already resolved price.

`<span is="inline-price" data-wcs-osi="r_JXAnlFI7xD6FxWKl2ODvZriLYBoSL701Kd1hRyhe8"></span>` should be sufficient. Please apply this to all `inline-price` and `checkout-link`